src/py/crankshaft/crankshaft/clustering/moran.py

```diff
@@ -11,7 +11,52 @@
 
 # High level interface ---------------------------------------
 
-def moran_local(subquery, attr, significance, num_ngbrs, permutations, geom_column, id_col, w_type):
+def moran(subquery, attr_name, permutations, geom_col, id_col, w_type, num_ngbrs):
+    """
+    Moran's I (global)
+    Implementation building neighors with a PostGIS database and Moran's I core clusters with PySAL.
+    Andy Eschbacher
+    """
+    qvals = {"id_col": id_col,
+             "attr1": attr_name,
+             "geom_col": geom_col,
+             "subquery": subquery,
+             "num_ngbrs": num_ngbrs}
+
+    q = get_query(w_type, qvals)
```

Comment:
Might be good to have a more descriptive name for get_query. How about construct_morans_query ? get_query reads like something we might have to use later on in a more general context